nginx/config.go

```diff
@@ -16,6 +16,13 @@ events {
 }
 
 http {
+	# basic settings
+	vhost_traffic_status_zone;
```

Comment:
Some vts reference here, for my own edification: https://github.com/vozlt/nginx-module-vts#vhost_traffic_status_zone